ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-popover.test.js

```diff
@@ -64,7 +64,7 @@ describe('AdvancedGasFeePopover', () => {
   it('should enable save button as input value is changed', () => {
     render();
     fireEvent.change(document.getElementsByTagName('input')[0], {
-      target: { value: 4 },
+      target: { value: 3 },
```

Comment:
Why did this need to change?

Reply:
With validation added a greater multiplier was required to  get correct value for save button to be enabled.